src/fontra/views/editor/edit-tools-pointer.js

```diff
@@ -293,6 +293,11 @@ export class PointerTool extends BaseTool {
   }
 
   async handleDragSelection(eventStream, initialEvent) {
+    const resizeSelectionViewStatus =
```

Comment:
Please open a second window, look at the user settings, then go back to the first and drag some points. Observe the "resize selection" checkbox go on and off.

I don't think this is the best way to do it. Isn't there a property you already use for the bounds resize handles that you can temporarily reset?

Reply:
Oh damn. Sorry. I totally ignored that. You're absolutely right. 
Sadly there is not property I can temporarily reset, but I added one. I hope that's better?!
Thanks for your quick feedback.